go/chat/localizer.go

```diff
@@ -1012,15 +984,9 @@ func (s *localizerPipeline) localizeConversation(ctx context.Context, uid gregor
 	case chat1.ConversationMembersType_TEAM:
 		// do nothing
 	case chat1.ConversationMembersType_IMPTEAMNATIVE, chat1.ConversationMembersType_IMPTEAMUPGRADE:
-		public := conversationLocal.Info.Visibility == keybase1.TLFVisibility_PUBLIC
-		resetUsernamesPegboard, err := s.getResetUsernamesPegboard(ctx, umapper, membersType, info, public)
```

Comment:
I don't really understand what was going on here or why it was unneeded.

Reply:
An unused feature